app/graphql/resolvers/comms-message.js

```diff
@@ -0,0 +1,11 @@
+import commsEventByPK from '../resolvers/queries/AHWP/get-comms-event-by-id.js'
+import commsEventByProperty from '../resolvers/queries/AHWP/get-by-property.js'
+
+const commsResolvers = {
```

Comment:
wasn't flagged in Sonar as an issue. Hmm, not sure if this brings more clarity, as comms-message is nested within resolvers folder which gives it a context. Will look into this

Reply:
Fair enough, think just because i've encountered that suggestion in the past from SonarCloud, it's worth considering it as a means of good practice. Have added a similar comment in other areas of this PR so would say maybe it's something to consider on a case by case basis.

Reply:
Renaming this to comms-resolvers